### PR TITLE
feat: style game board and lobby

### PR DESCRIPTION
**Pull request by masonwongcs**

### CODE REVIEW

File: src/components/Game/GameBoard.module.sass [hunk 1]
```diff
@@ -5,6 +5,7 @@
   width: 100%
   background-color: mediumseagreen
   display: flex
+  transform: rotateX(35deg)
```
Comment (ginsan95) [inline src/components/Game/GameBoard.module.sass]: Sorry bro, I don't think I want the rotate with perspective. It actually makes things harder to see and ruin some calculation as well. Not mobile friendly too.

File: src/components/Room/GameRoom.tsx [hunk 2]
```diff
@@ -197,38 +197,42 @@ const GameRoom: React.FunctionComponent = () => {
...
-                                style={
-                                    player.name === myName
-                                        ? { fontWeight: 'bold' }
-                                        : undefined
-                                }
+            <div className={styles.wrapper}>
+                <div className={styles.title}>{renderTitleContent()}</div>
+                <div className={styles.list_container}>
+                    {players.map((player) => (
+                        <div className={styles.list_container_item}>
```
Comment (ginsan95) [inline src/components/Room/GameRoom.tsx]: Missing key

File: src/components/Room/GameRoom.tsx [hunk 3]
```diff
@@ -197,38 +197,42 @@ const GameRoom: React.FunctionComponent = () => {
...
+                            <div
+                                className={styles.player_avatar}
+                                style={{
+                                    backgroundImage: `url(https://avatars.dicebear.com/api/micah/${player.name}.svg)`,
+                                }}
+                            />
+                            <div
+                                className={`${styles.player_name} ${
+                                    player.name === myName ? styles.player_name_self : ''
+                                }`}
```
Comment (ginsan95) [inline src/components/Room/GameRoom.tsx]: Maybe can use `cssUtils.joinClassName` for this

File: src/variable.sass [hunk 4]
```diff
@@ -0,0 +1,2 @@
+$gradient-bg: linear-gradient(to bottom, #11998e, #38ef7d)
```
Comment (ginsan95) [inline src/variable.sass]: I feel the bottom colour is too bright. Can we tone down the diff a little?

Comment (masonwongcs) [reply src/variable.sass]: Ok bro, let me adjust a little bit.

File: src/components/Room/GameRoom.tsx (hunk 3 above)
Comment (masonwongcs) [reply src/components/Room/GameRoom.tsx]: Okay

File: src/components/Game/GameBoard.module.sass (hunk 1 above)
Comment (masonwongcs) [reply src/components/Game/GameBoard.module.sass]: Yea, let me use the 2D view instead.

**Review by ginsan95 — APPROVED** [review]

**Pull request merged**